### PR TITLE
Documentation and other tweaks before release

### LINKED ISSUE #374: Web UI typo in Legend
`The image comparison metrics used to compare variant and referencedisagree`

### PR DESCRIPTION
**Pull request by paulthomson**
Fix #374

### CODE REVIEW

File: docs/glsl-fuzz-develop.md [hunk 1]
```diff
@@ -158,7 +201,15 @@ cd gles-worker/
 
 ## Vulkan worker developer documentation
```
Comment (afd) [inline docs/glsl-fuzz-develop.md]: Should we call this "Legacy Vulkan worker developer documentation"?

File: docs/glsl-fuzz-walkthrough.md [hunk 2]
```diff
@@ -544,31 +538,47 @@ in the next section,
...
 From here,
 we can queue a reduction of the variant shader
 to find a smaller, simpler shader
 that still exposes the bug.
-The default reduction settings are sufficient, so just click
+The "fuzzy diff" algorithm is selected by default,
+which will work in this case,
+as both metrics detected the image as different from the reference.
+You can select the "HISTOGRAM_CHISQR" metric from the "Comparison metric" dropdown
+to switch the histogram comparison metric;
```
Comment (afd) [inline docs/glsl-fuzz-walkthrough.md]: to switch -> to switch to

File: docs/glsl-fuzz-walkthrough.md [hunk 3]
```diff
@@ -544,31 +538,47 @@ in the next section,
...
+you can then adjust the threshold based on the histogram distance reported
+higher up the page, although we find that 100.0 usually works,
+which is the default.
+In this case,
```
Comment (afd) [inline docs/glsl-fuzz-walkthrough.md]: Saying "In this case, the ..." sounds like they'll actually be reducing this bug, but they won't as it's just a bug we found.

File: docs/glsl-fuzz-develop.md (hunk 1 above)
Comment (paulthomson) [reply docs/glsl-fuzz-develop.md]: Done.

**Pull request merged**